config_loader.py

```diff
@@ -0,0 +1,72 @@
+import yaml
+
+'''
+Configuration keys enumeration
+'''
+
+
+class ConfigKeys:
+    DESKS_ARRAY = "desks"
+    DESK_ID = "id"
+    DESK_NAME = "name"
+    DESK_SENSOR_PIN = "sensor_pin"
+    DEBUG_MODE = "debug_mode"
+
+
+''' 
+Singleton configuration instance
+'''
+__config = None
+
+'''
+This class is responsible for loading the configuration of the whole program, and
+should be responsible for parsing the data so that it is fairly easy to access what we need when using this class
+within other classes
+'''
+
+
+class YAMLConfigLoader:
+    __yaml_file = ""
+    __property_dictionary = {}
+
+    '''
+    Initialize Config Loader with config.yaml as default file
+    '''
+    def __init__(self, yaml_file):
+        self.__yaml_file = yaml_file
+        self.reload_config()
+
+    '''
+    Loads YAML config file. Could potentially change this to SQLite in future,
+    so that a user cant go in and just change values (if this is the case, it should still be managed by us)
+    '''
+    def reload_config(self):
```

Comment:
Why is this called `reload_config` as opposed to just `load_config`?

Reply:
I originally had a load/reload method doing slightly different things, and ended up keeping only the reload method, forgot to rename! Will do